NinchatSDKSwift/Resources/en.lproj/Localizable.strings

```diff
@@ -15,4 +15,5 @@
 "Disagree" = "Disagree";
 "Strongly disagree" = "Strongly disagree";
 "Permission denied" = "Permission denied.";
-"Update Settings" = "Tap to update Settings.";
\ No newline at end of file
+"Update Settings" = "Tap to update Settings.";
+"WebRTC pickup fail" = "failed to send WebRTC pickup message";
```

Comment:
If this message is displayed to the end user when there is a problem initializing video, then maybe a bit more user friendly error could be used? "Problem when starting video chat" or something?

Reply:
Yes, you're right. I'll update them